src/panels/SkyBrowserPanel/hooks.ts

```diff
@@ -0,0 +1,223 @@
+import { useEffect } from 'react';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import {
+  subscribeToSkyBrowser,
+  unsubscribeToSkyBrowser
+} from '@/redux/skybrowser/skybrowserMiddleware';
+import {
+  setActiveImage,
+  setImageCollectionData
+} from '@/redux/skybrowser/skybrowserSlice';
+
+export function useGetWwtImageCollection() {
+  const luaApi = useOpenSpaceApi();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    async function getWwtListOfImages() {
+      if (!luaApi || !luaApi.skybrowser) {
+        return;
+      }
+      const imgData = await luaApi.skybrowser.listOfImages();
+      if (imgData) {
+        const images = Object.values(imgData);
+        if (images.length !== 0) {
+          const imgDataWithKey = images.map((image) => ({
+            ...image,
+            key: image.identifier
+          }));
+          dispatch(setImageCollectionData(imgDataWithKey));
+        }
+      } else {
+        throw new Error('No AAS WorldWide Telescope images!');
+      }
```

Comment:
This already simplifies it a little
```suggestion
      if (!imgData) {
        throw new Error('No AAS WorldWide Telescope images!');
      }
      const images = Object.values(imgData);
      if (images.length > 0) {
        const imgDataWithKey = images.map((image) => ({
          ...image,
          key: image.identifier
        }));
        dispatch(setImageCollectionData(imgDataWithKey));
      }
      
```